Updated README.md (#426)
XengShi/materialYouNewTab: README.md

     - Ensure you’ve cloned the repository or downloaded the ZIP file and extracted it.

   - **Open Extensions Page**:
     Open your preferred browser and enter the following text in the address bar:

     - ![Chrome](https://img.icons8.com/color/20/000000/chrome--v1.png) Chrome: `chrome://extensions`

     - ![Edge](https://img.icons8.com/?size=20&id=dGm9KIZPpukc&format=png&color=000000) Edge: `edge://extensions`

     - ![Brave](https://img.icons8.com/color/20/000000/brave-web-browser.png) Brave: `brave://extensions`

>     

Reviewer: Unintended indentation , sorry, my bad

   - [Mozilla Add-ons link](https://addons.mozilla.org/en-US/firefox/addon/material-you-newtab/)

>    <img src="https://i.postimg.cc/bPW2fHX7/FireFox.png" alt="Screenshot" width="569">

Reviewer: ```suggestion
<img src="https://i.postimg.cc/bPW2fHX7/FireFox.png" alt="Screenshot" width="569">
```

   - [Mozilla Add-ons link](https://addons.mozilla.org/en-US/firefox/addon/material-you-newtab/)

   

>    <img src="https://i.postimg.cc/bPW2fHX7/FireFox.png" alt="Screenshot" width="569">

Reviewer: Maybe delete extra line 125 and indentation at line 126.. 
Else no top margin on image, after Mozilla Add-ons link

Author: The image and the link gets together without that space

Reviewer: It is like this rn (no space between link and image)
![IMG_20250105_094042](https://github.com/user-attachments/assets/bf5ec025-9103-43a4-ae88-e50382e81803)

There are two newline, deleting one line wont cause them getting together 
![IMG_20250105_094107](https://github.com/user-attachments/assets/25d10afc-74d2-40e4-8c56-855f5cdd5221)

This way there is margin between link and image 
![IMG_20250105_094123](https://github.com/user-attachments/assets/143e2482-5ba5-4fc4-969b-1126f5386192)

Author: So what? would i remove that extra space?

Reviewer: I think you should.

---

> #### ![Opera](https://img.icons8.com/color/20/000000/opera--v1.png) Opera or ![Opera GX](https://img.icons8.com/color/20/000000/opera-gx.png) Opera GX

Author: Later. Mybe tomorrow. Or you are allowed to do the changes if you want by opening a pr on my repo 🙃

> #### ![Opera](https://img.icons8.com/color/20/000000/opera--v1.png) Opera or ![Opera GX](https://img.icons8.com/color/20/000000/opera-gx.png) Opera GX

Reviewer: I am not able to create pr on your repo after adding the rules.

also on XengShi's repo, I cannot create automatic branch: patch-1, patch-2, ...

> #### ![Opera](https://img.icons8.com/color/20/000000/opera--v1.png) Opera or ![Opera GX](https://img.icons8.com/color/20/000000/opera-gx.png) Opera GX

Author: Odd... I don’t think that the rules will block you from creating any pr...

> #### ![Opera](https://img.icons8.com/color/20/000000/opera--v1.png) Opera or ![Opera GX](https://img.icons8.com/color/20/000000/opera-gx.png) Opera GX

Reviewer: but it is :(

7. **Wallpaper Management**:
   - Users can upload their own wallpapers via the **new wallpaper management section** in the menu settings.
   - The **Random Image feature** provides users with a new wallpaper every day, updating automatically at the start of each day. Images are sourced from [Lorem Picsum](https://picsum.photos).

8. **Backup System**:
   - The menu settings include a backup system that allows users to save and restore their customized settings.
   - This feature is detailed with screenshots, showing how users can manage backups effectively.

9. **Bookmarks**:
   - The extension allows users to manage their bookmarks directly.
   - Users can delete any bookmarks they no longer need, ensuring a clean and organized space.
   - A **view as selector** lets users toggle between a **grid** and **list layout**, allowing them to display their bookmarks as they prefer.

10. **To-Do List**:
    - The extension features a **to-do list** to help users remember important tasks.

>     - Users can pin specific tasks using the **pin button**, ensuring those tasks remain at the top until the pin is removed or the task is deleted. This feature makes it easy to keep essential items front and center.

Author: ohh..

## ❓ Issues and Support

- If you encounter any issues or bugs, feel free to open an issue on [GitHub](https://github.com/XengShi/materialYouNewTab/issues).
- For general questions or support, you can contact the repository maintainers through the **Telegram group**: [Join here](https://t.me/+i-ekvlM7yoU1Yjc1).

## 🛡 Privacy Policy

> Material You NewTab respects your privacy. For more details, read our [Privacy Policy](https://github.com/XengShi/materialYouNewTab/privacy-policy).

Reviewer: ```suggestion
Material You NewTab respects your privacy. For more details, read our [Privacy Policy](https://xengshi.github.io/materialYouNewTab/privacy-policy.html).
```

## 📜 License

Copyright (C) 2023-2025 XengShi

This project is licensed under the GNU General Public License v3.0 (GPL-3.0). See the [LICENSE](https://github.com/XengShi/materialYouNewTab/blob/main/LICENSE) file for details.

> [GitHub](https://github.com/XengShi) &nbsp; [Telegram](https://t.me/+i-ekvlM7yoU1Yjc1) &nbsp; [Email](mailto:xengshi@outlook.com)

Reviewer: ```suggestion
[GitHub](https://github.com/XengShi) &nbsp; [Telegram](https://t.me/xengshi) &nbsp; [Email](mailto:xengshi@outlook.com)
```
there are Xengshi's contact, maybe leave it.. already mentioned group link above and in feedback page
